roles/system/templates/deployments/system-sidekiq-deployment.yaml

```diff
@@ -0,0 +1,465 @@
+          image: "{{ image.name | default(image_name) }}:{{ image.tag | default(image_tag) }}"
+          imagePullPolicy: "{{ image.pullPolicy | default(image_pull_policy) }}"
+          args:
+            - rake
+            - sidekiq:worker
+            - RAILS_MAX_THREADS=25
+        ## container > system-sidekiq > ports
+          ports:
+            - containerPort: 9394
+              name: metrics
+              protocol: TCP
+        ## container > system-sidekiq > resources
+          resources:
+            requests:
+              memory: "{{ sidekiq.resources.requests.memory | default(sidekiq_resources_requests_memory) }}"
+              cpu: "{{ sidekiq.resources.requests.cpu | default(sidekiq_resources_requests_cpu) }}"
+            limits:
+              memory: "{{ sidekiq.resources.limits.memory | default(sidekiq_resources_limits_memory) }}"
+              cpu: "{{ sidekiq.resources.limits.cpu | default(sidekiq_resources_limits_cpu) }}"
+          livenessProbe:
+            httpGet:
+              path: /metrics
+              port: 9394
+              scheme: HTTP
+            initialDelaySeconds: {{ sidekiq.liveness_probe.initial_delay_seconds | default(sidekiq_liveness_probe_initial_delay_seconds) }}
+            timeoutSeconds: {{ sidekiq.liveness_probe.timeout_seconds | default(sidekiq_liveness_probe_timeout_seconds) }}
+            periodSeconds: {{ sidekiq.liveness_probe.period_seconds | default (sidekiq_liveness_probe_period_seconds) }}
+            successThreshold: {{ sidekiq.liveness_probe.success_threshold | default(sidekiq_liveness_probe_success_threshold) }}
+            failureThreshold: {{ sidekiq.liveness_probe.failure_threshold | default(sidekiq_liveness_probe_failure_threshold) }}
+          readinessProbe:
+            httpGet:
+              path: /metrics
+              port: 9394
+              scheme: HTTP
+            initialDelaySeconds: {{ sidekiq.readiness_probe.initial_delay_seconds | default(sidekiq_readiness_probe_initial_delay_seconds) }}
+            timeoutSeconds: {{ sidekiq.readiness_probe.timeout_seconds | default(sidekiq_readiness_probe_timeout_seconds) }}
+            periodSeconds: {{ sidekiq.readiness_probe.period_seconds | default (sidekiq_readiness_probe_period_seconds) }}
+            successThreshold: {{ sidekiq.readiness_probe.success_threshold | default(sidekiq_readiness_probe_success_threshold) }}
+            failureThreshold: {{ sidekiq.readiness_probe.failure_threshold | default(sidekiq_readiness_probe_failure_threshold) }}
+        ## container > system-sidekiq > volumeMounts
+          volumeMounts:
+            - mountPath: /tmp
+              name: system-tmp
+            - mountPath: /opt/system-extra-configs
+              name: system-config
+              readOnly: true
+        ## container > system-sidekiq > env
+          env:
+            - name: THINKING_SPHINX_ADDRESS
+              value: {{ sphinx.env.thinking.endpoint | default("system-sphinx") }}
```

Comment:
```suggestion
              value: {{ sphinx.env.thinking.endpoint | default(sphinx_env_thinking_endpoint) }}
```

Reply:
This was intentional, as by default will use the `system-sphinx` service (the service object name) created by the SaaS operator.